osu.Game.Rulesets.Osu/Mods/OsuModWiggle.cs

```diff
@@ -42,30 +57,30 @@ private void drawableOnApplyCustomUpdateState(DrawableHitObject drawable, ArmedS
             Random objRand = new Random((int)osuObject.StartTime);
 
             // Wiggle all objects during TimePreempt
-            int amountWiggles = (int)osuObject.TimePreempt / wiggle_duration;
+            int amountWiggles = (int)osuObject.TimePreempt / WiggleDuration.Value;
 
             void wiggle()
             {
                 float nextAngle = (float)(objRand.NextDouble() * 2 * Math.PI);
-                float nextDist = (float)(objRand.NextDouble() * wiggle_strength);
-                drawable.MoveTo(new Vector2((float)(nextDist * Math.Cos(nextAngle) + origin.X), (float)(nextDist * Math.Sin(nextAngle) + origin.Y)), wiggle_duration);
+                float nextDist = (float)(objRand.NextDouble() * WiggleStrength.Value * 7);
```

Comment:
This changes the default from `wiggle_strength` (which was 10) to 7. Was this intended?

Reply:
yes, in the interest of centering the default value at 1. 20 as the max (2x the base strength ) felt overkill upon playtesting and since wiggle is a new lazer mod  and the value seemed arbitrary i recentered it around 7

Reply:
Do keep in mind that this will break all replays to date. Although with the way wiggle is implemented (unseeded movement) replays are kind of doomed regardless, so I'm willing to overlook this for now.